pkgs/top-level/all-packages.nix

```diff
@@ -967,6 +967,8 @@ in
 
   aws-google-auth = python3Packages.callPackage ../tools/admin/aws-google-auth { };
 
+  aws-nuke = python3Packages.callPackage ../tools/admin/aws-nuke { };
```

Comment:
Should be the regular `callPackage`, no?

Reply:
Hah. My copy-paste is leaking!

Reply:
Fixed :sweat_smile: